volumes/rescu/README.md

```diff
@@ -1,45 +1,316 @@
+>- ATM Active Traffic Management team uses the data but they do not deal with the hardware or functionality of the data. 
+>- ITS Central: Transnomis (Simon) via Black/Mac is responsible.
+>- Oracle DB Server: Jim is responsible as he functions as the owner of the dataset.
+
+### 2- How often is data maintained? Are there monitoring mechanisms when parts of the data flow is not working?	
+>Unsure. 
+### 3- Is there a chain/process for when equipment goes wrong/breaks?	
+> Unsure.
+### 4- What is the process that needs to be undertaken when equipment goes wrong/breaks? 	> Unsure.
+### 5-Who should be notified if something goes wrong/ there are changes to data? 	
+>Data specialists within D&A. Data users within TMC. 
+
+### 5- Are external contractors or consultants responsible for maintenance? Who is the City staff contact who works for that vendor?	
+>ATM is leased to a contractor in the traffic control room. Contractor runs and manages the work orders. 
+ 
+
+
+
+# DATA-SPECIFIC INFORMATION FOR:
+## Schema
+Data is stored in RESCU schema.
+## Number of variables: 
+
+
+>`raw_20sec` table:
+"datetime_bin",
+"detector_id",
+"lane_no",
+"volume",
+"occupancy",
+"speed",
+"uid"
 
-This [script](rescu_pull.py) is located in the terminal server and takes in three variables which are `start_date`, `end_date` (exclusive) and `path`. It is a job that runs on a daily basis in the morning and imports the latest 15-minute volume file from a particular drive into our RDS. By default, the `start_date` and `end_date` are the beginning and the end of the day before (today - 1 day and 12:00 today respectively). However, the user can also specify the date range from which the data should be pulled to ensure that this process can be used for other applications too. This script is automated to run daily on the terminal server to ingest a day worth of data collected from the day before. The steps are as followed:
+>`raw_15min` table:
+"dt", 
+"raw_info", 
+"raw_uid"
 
-1) The script reads information from a .rpt file and then inserts the date into a table named `rescu.raw_15min`. The table has the following information
+>`volumes_15min` table:
+"detector_id",
+"datetime_bin",
+"volume_15min",
+"arterycode",
+"volume_uid",
 
-|raw_uid|dt|raw_info|
-|--|--|--|
-|12852|2020-01-13|1700 - 1715 de0010deg    633|
-|12853|2020-01-13|1700 - 1715 de0010der    62|
 
-2) There is also a trigger function named [`rescu.insert_rescu_volumes()`](create-trigger-function-populate_volumes_15min.sql) which processes the newly added data from `rescu.raw_15min` and inserts the processed data into the table `rescu.volumes_15min`. All information from `raw_info` in the raw table is then processed into the 3 columns which are `detector_id`, `datetime_bin` and `volume_15min` whereas `aretrycode` is taken from the table `rescu.detector_inventory` by matching them using `detector_id`. The table `rescu.volumes_15min` has the following information
+## Missing data: 
+> TBA
```

Comment:
Let's hide (or "comment out") headings that don't have info. They'll stay in the template but they won't be visible in the preview.
To comment out multi-line stuff do this:

```
<!--- # I am a heading
I am the information that goes under the heading --->

```